<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiTL2PowerSupply</class>
 <widget class="QMainWindow" name="uiTL2PowerSupply">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TL2 Power Supplies</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="GBBendingMagnets_2">
        <property name="styleSheet">
         <string notr="true">QGroupBox {
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
        </property>
        <property name="title">
         <string>Correctors</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="layoutCR">
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>47</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="GBBendingMagnets">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
      </property>
      <property name="title">
       <string>Bending Magnets</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="layoutBM">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="GBQuadropoles">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
      </property>
      <property name="title">
       <string>Quadropoles</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="layoutQuads">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="GBThickSeptum">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
      </property>
      <property name="title">
       <string>Thick Septum</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="layoutTHS"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="GBClose">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
 	background-color: rgb(255, 255, 255);
     border: 1px solid black;
     border-radius: 5px;
     margin-top: 0.5em;
 }
 
 QGroupBox::title {
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>144</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder>GBBendingMagnets</zorder>
   <zorder>GBClose</zorder>
   <zorder>GBThickSeptum</zorder>
   <zorder>GBQuadropoles</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
